Src/Tracer.InterfacedObject.pas

```diff
@@ -28,16 +34,22 @@ implementation
 
 uses
 {$IFDEF DEBUG}
-  Winapi.Windows,
+  Windows,
   Tracer.Logger,
 {$ENDIF}
-  System.SysUtils;
+  SysUtils;
 
 { TTracerInterfacedObject }
 
 function TTracerInterfacedObject._AddRef: Integer;
 begin
-  Result := AtomicIncrement(FRefCount);
+  FCriticalSection.Enter;
```

Comment:
The critical section is overkill here. There is "InterlockedIncrement" on older Delphi versions. Can you please change this?